app/services/idv/analytics_events_enhancer.rb

```diff
@@ -50,7 +50,28 @@ def self.included(_mod)
     def common_analytics_attributes
       {
         proofing_components: proofing_components,
-      }
+        active_profile_idv_level: active_profile&.idv_level,
+        pending_profile_idv_level: pending_profile&.idv_level,
+        profile_history: profile_history,
+
+      }.compact
```

Comment:
style nit: remove this blank line?
```suggestion
        profile_history: profile_history,
      }.compact
```